packages/components/psammead-grid/src/index.test.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { Grid } from '.';
+import { ExampleParagraph, ExampleFigure } from './helpers';
+
+describe('Grid component', () => {
+  shouldMatchSnapshot(
+    'should render Grid wrapper with Grid',
+    <Grid
+      columns={{ group3: 6, group4: 8, group5: 20 }}
```

Comment:
Great suggestion. I'd suggest these not be within this Psammead component, but be in Simorgh, since they're very specific to the page layout of Simorgh articles/ front pages / etc.